### PR TITLE
Property listing branch implemented with home listings

### PR DESCRIPTION
**Pull request by asiedu-george**
This branch contains featured contents of dummy home listings to be displayed on the landing page

### CODE REVIEW

File: src/app/landing-page/listing/listing.component.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            Discover our hand-picked selection of the most desirable properties on the market. 
+            Each featured property offers unique charm, exceptional amenities, <br> and prime locations. 
+            Explore these top listings to find your dream home today. 
+        </p>
+   </div>
+   <div class="card">
+        <app-listing-card/>
+        <app-listing-card/>
+        <app-listing-card/>
+        <app-listing-card/>
```
Comment (Kwabena-Owusu-Achiaw) [inline src/app/landing-page/listing/listing.component.html:14]: use an *ngFor directive to handle this.

File: src/app/shared/listing-card/listing-card.component.scss [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            cursor: pointer;
+            font-weight: 600;
+            font-size: 0.65rem;
+            transition: .3s ease-in;
+            text-transform: uppercase;
+  
+            &:hover {
+                background-color: #fff;
+                color: #1A1A1A;
+                border: 2px solid #1A1A1A;
```
Comment (Kwabena-Owusu-Achiaw) [inline src/app/shared/listing-card/listing-card.component.scss:37]: This color is defined at serval places. declare is color in your global style sheet and use it throughout the application

File: src/app/landing-page/listing/listing.component.html (hunk 1 above)
Comment (asiedu-george) [reply src/app/landing-page/listing/listing.component.html:14]: done

File: src/app/shared/listing-card/listing-card.component.scss (hunk 2 above)
Comment (asiedu-george) [reply src/app/shared/listing-card/listing-card.component.scss:37]: done

**Pull request merged**